mod.test.ts

```diff
@@ -1648,6 +1651,11 @@ Deno.test("ensure KillSignalController readme example works", async () => {
   assert(endTime - startTime < 1000);
 });
 
+Deno.test("should support empty quoted string", async () => {
+  const output = await $`echo '' test ''`.text();
+  assertEquals(output, " test ");
+});
```

Comment:
I added a basic test. In the future, I recommend doing this because it helps prevent against regressions.